test/performance/array/distCreate.cc-numlocales

```diff
@@ -0,0 +1 @@
+16
```

Comment:
I think all the other cc tests use 4 locales. I'm not sure if 16 will cause problems, but if 4 will work let's go with that.